terraform/10-video-hearing-wowza/modules/wowza/vm_automation.tf

```diff
@@ -1,7 +1,6 @@
 locals {
   cert_env            = var.environment == "prod" ? "" : "${var.environment}-"
   wowza_domain        = "vh-wowza.${local.private_dns_zone}"
-  wowza_public_domain = "vh-wowza-public.${local.private_dns_zone}"
```

Comment:
why are we removing the public domain?

Reply:
following the pattern for all other apps.

same domain name, on VPN resolves to private IP, off VPN resolves to CNAME of Public IP DNS Name.

This caused a bit of confusion yesterday, this is why we are changing it to follow the pattern of all our other apps.